### PR TITLE
Update cosmos-compose.json

### PR DESCRIPTION
**Pull request by natevns08**
update cosmos-compose for immich 1.91.0 update [full change log here](https://github.com/immich-app/immich/releases)
 - Remove typesense
 - Change postgres image

### CODE REVIEW

**Comment by azukaar** [comment]
Please test it before you make a PR

File: servapps/Immich/cosmos-compose.json [hunk 1]
```diff
@@ -64,8 +64,7 @@
       ],
       "depends_on": [
         "{ServiceName}-redis",
-        "{ServiceName}-database",
-        "typesense"
+        "{ServiceName}-database",
```
Comment (Kawanaao) [inline servapps/Immich/cosmos-compose.json:67]: Need to remove trailing coma

File: servapps/Immich/cosmos-compose.json [hunk 2]
```diff
@@ -127,8 +126,7 @@
...
```
Comment (Kawanaao) [inline servapps/Immich/cosmos-compose.json:129]: Need to remove trailing coma

**Review by Kawanaao — CHANGES_REQUESTED** [review]
You need to remove some commas and then everything will work

**Pull request merged**